lib/const.rb

```diff
@@ -115,8 +117,12 @@
 CREW_CACHE_BUILD        = ENV.fetch('CREW_CACHE_BUILD', '0').eql?('1')
 CREW_CACHE_FAILED_BUILD = ENV.fetch('CREW_CACHE_FAILED_BUILD', '0').eql?('1')
 
-# Set CREW_NPROC from environment variable or `nproc`
-CREW_NPROC = ENV.fetch('CREW_NPROC', `nproc`.chomp)
+# Set CREW_NPROC from environment variable, `distcc -j`, or `nproc`.
+CREW_NPROC = if File.file?("#{CREW_PREFIX}/bin/distcc")
+               ENV.fetch('CREW_NPROC', `distcc -j`.chomp)
+             else
+               ENV.fetch('CREW_NPROC', `nproc`.chomp)
+             end
```

Comment:
What's the difference?

Reply:
With multiple distcc hosts, the number can be much higher than what nproc gives, as work can be offloaded to other machines...